libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -851,6 +866,59 @@ AP_GPS_UBLOX::_parse_gps(void)
         state.horizontal_accuracy = 0;
 #endif
         break;
+
+    case MSG_NAV_RELPOSNED: {
+        Debug("MSG_NAV_RELPOSNED RTK status Received");
+
+        state.rtk_age_ms    = 0xFFFFFFFF;
+
+    	state.rtk_baseline_x_mm = _buffer.rtk.rel_pos_n_cm*10+_buffer.rtk.rel_pos_hp_n_mm;
```

Comment:
Also note that the HP mm components are actually scaled by 10 (IE reported in a range of -99 to 99) you need to divide them by 10 before adding)